beprof/profile.py

```diff
@@ -144,11 +146,14 @@ def normalize(self, dt):
         except RuntimeWarning as e:
             logger.error('in normalize(). self class is {0}, dt={1}'.format(self.__class__, dt))
             raise Exception("Scaling factor error:\n" + str(e))
-        self.y /= ave
+        self.y = self.y / ave
 
     def __str__(self):
         logger.info('Running {0}.__str__'.format(self.__class__))
-        ret = super().__str__()
+        # todo: investigate why super().__str__() errors in python 2.7
+        #       after change of implementation of __new__
+        ret = curve.Curve.__str__(self)
+        # ret = super(curve.Curve).__str__()
```

Comment:
Remove it, also "investigate TODO" can directly go as new issue, instead of dirty comment.